index.js

```diff
@@ -1,60 +1,65 @@
-let books = [];
-
-function Book(title, author) {
-  this.title = title;
-  this.author = author;
-}
-
-const setStorage = (books) => {
-  const dataBook = JSON.stringify(books);
-  localStorage.setItem('data-books', dataBook);
-};
-
-const removeBook = (title, author) => {
-  books = books.filter((book) => book.title !== title || book.author !== author);
-};
+/* eslint-disable class-methods-use-this */
+/* eslint-disable max-classes-per-file */
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+    this.books = [];
+    this.booksContainer = document.getElementById('books-container');
+  }
 
-const displayBook = (book) => {
-  const booksContainer = document.getElementById('books-container');
+  storeBook(book) {
+    localStorage.setItem('data-books', JSON.stringify(book));
+  }
 
-  const bookCard = document.createElement('div');
-  bookCard.style.borderBottom = '1px solid #000';
+  getBooks() {
+    if (!localStorage.getItem('data-books')) {
+      this.books = [];
+    } else {
+      this.books = JSON.parse(localStorage.getItem('data-books'));
+    }
+    return this.books;
+  }
 
-  const pTitle = document.createElement('p');
-  pTitle.textContent = book.title;
+  addBook(book) {
+    const recoveredBooks = this.getBooks();
+    recoveredBooks.push(book);
+    this.storeBook(recoveredBooks);
+  }
 
-  const pAuthor = document.createElement('p');
-  pAuthor.textContent = book.author;
+  removeBook(title, author) {
+    this.books = this.books.filter((book) => book.title !== title || book.author !== author);
+  }
```

Comment:
- [ ] [OPTIONAL] Your `removeBook` function works perfectly 👏 but I would like to suggest that you consider using a unique id property for each book since it would make it easier to remove books with duplicate values and you don't have to pass in `title` and `author` to your function anymore 👍